### DISCUSSION TITLE
Support for other-than-battery devices

### DISCUSSION THREAD

**Discussion by KrzysztofHajdamowicz** [andrew-codechimp/HA-Battery-Notes#2227] [Ideas]
Like, for example, I want to track filter level in fleet of my Xiaomi Air Purifiers.

HA Integration provides device with filter level entity, so it should be quite easy to add it to Battery Notes.
I can provide Pull Request with that.

**Comment by andrew-codechimp** [comment]
Hi, this has been suggested before for things like printer ink.
I don't want to extend Battery Notes beyond batteries, but there's always scope for someone to create an integration to track other maintenance type records.
